quixstreams/kafka/consumer.py

```diff
@@ -29,9 +29,13 @@
 
 
 def _default_error_cb(error: KafkaError):
-    logger.error(
-        f"Kafka consumer error: {error.str()} (code={error.code()})",
-    )
+    error_code = error.code()
+    if error.fatal():
```

Comment:
@tim-quix are you sure what `.fatal()` means in this context? 
From the docs, it seems to be relevant to Kafka Transactions.

Could you pls confirm that `.fatal()` does what we expect here?

Reply:
could not confirm one way or the other, so went ahead and removed for now.